benches/common/test_vectors.rs

```diff
@@ -0,0 +1,37 @@
+//! Benchmark parameters
+//! The parameters chosen correspond to cases of interest for Cardano's Peras and Leios protocols.
+
+pub mod centralized {
+    use super::super::criterion_helpers::centralized::BenchParam;
+
+    /// This case corresponds to the minimum security requirements with optimistic set_size
+    const LOW_PARAM: BenchParam = BenchParam {
```

Comment:
Suppose I want to run a benchmark with particular n_p, n_f and |S_p| (size of prover's input). Currently, I would need to think hard to figure out how to correctly set the percentages and `set_cardinality` here.

A better option could be to have
```
    pub struct BenchParam {
        pub lambda_sec: f64,
        pub lambda_rel: f64,
        pub set_cardinality: u64,
        pub set_size: u64,
        pub lower_bound: u64,
    }
```
and have utility functions in this file that generate these structs based on percentages. There would be no low / mid / high cases in `criterion_helpers.rs` `benchmarks()`, instead they would be generated here.

What do you think?